diddiparser2/parser.py

```diff
@@ -240,6 +241,9 @@ def execute_func(self, line):
                     else:
                         self.last_value = Null()
                 return None
+            except DSError:
+                # we *should* fail here
+                raise

```

Comment:
Just a little note: I made this to avoid issues with functions. Just ignore it :p

Reply:
Ah, this produced issues. Let's fix it.